src/main/java/com/todolist/app/servlet/NewTaskServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.todolist.app.servlet;
+
+
+import com.todolist.app.model.Item;
+import com.todolist.app.model.Priority;
+import com.todolist.app.model.Status;
+import com.todolist.app.model.Tag;
+import com.todolist.app.service.ToDoStoreSingleton;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@WebServlet(name = "NewTaskServlet", value = "/new-task")
+public class NewTaskServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.sendRedirect(request.getContextPath() + "/jsp/new-task.jsp");
```

Comment:
Can we use forward as it's done for ToDoServlet. It won't change the url in the browser after redirect. This is important.

https://files.fm/u/qgfdvrddx